app/src/main/java/ch/epfl/sdp/appart/user/AppUser.java

```diff
@@ -136,6 +144,14 @@ public void setProfileImage(String img) {
         this.profilePicture = img;
     }
 
+    /**
+     * removes the profile picture, thus sets the default user gender icon
+     */
+    @Override
+    public void removeProfileImage() {
+        this.profilePicture = null;
+    }
```

Comment:
We have to remember to modify this so that it removes the image also from the firestore user

Reply:
Sure, good point!